### PR TITLE
Remove privacy for reported media

### PR DESCRIPTION
**Pull request by yahilmadakiya**
When the user edits the media and if the same media is already reported by the other user then any changes on `edit page`, it should not automatically changed to approval.

**Fixed**.

### CODE REVIEW

**Comment by yahilmadakiya** [comment]
@mangesh 
This issue has been fixed mentioned on community :point_right:  http://community.rtcamp.com/t/buddypress-media-moderation-bug/7941

File: app/main/controllers/template/rtmedia-functions.php [hunk 1]
```diff
@@ -2487,10 +2487,14 @@ function rtmedia_edit_media_privacy_ui() {
 	$privacymodel = new RTMediaPrivacy( false );
 	$privacy      = $privacymodel->select_privacy_ui( $echo = false );
 
-	if ( $privacy  && empty( $comment_media ) ) {
-		return "<div class='rtmedia-edit-privacy rtm-field-wrap'><label for='privacy'>" . esc_html__( 'Privacy : ', 'buddypress-media' ) . '</label>' . $privacy . '</div>';
-	}
 
+	if ( isset( $rtmedia_query->media ) && is_array( $rtmedia_query->media ) && isset( $rtmedia_query->media['0'] ) ) {
+		if ( isset( $rtmedia_query->media['0']->privacy ) && $rtmedia_query->media['0']->privacy != '80' ) {
+			if ( $privacy  && empty( $comment_media ) ) {
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: seeing two spaces after `$privacy`

File: app/main/controllers/template/rtmedia-functions.php [hunk 2]
```diff
@@ -2487,10 +2487,14 @@ function rtmedia_edit_media_privacy_ui() {
 	$privacymodel = new RTMediaPrivacy( false );
 	$privacy      = $privacymodel->select_privacy_ui( $echo = false );
 
-	if ( $privacy  && empty( $comment_media ) ) {
-		return "<div class='rtmedia-edit-privacy rtm-field-wrap'><label for='privacy'>" . esc_html__( 'Privacy : ', 'buddypress-media' ) . '</label>' . $privacy . '</div>';
-	}
 
+	if ( isset( $rtmedia_query->media ) && is_array( $rtmedia_query->media ) && isset( $rtmedia_query->media['0'] ) ) {
+		if ( isset( $rtmedia_query->media['0']->privacy ) && $rtmedia_query->media['0']->privacy != '80' ) {
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: add comment above
`//todo strict standard error`

File: app/main/controllers/template/rtmedia-functions.php [hunk 3]
```diff
@@ -2824,12 +2828,12 @@ function show_rtmedia_like_counts() {
  *
  * @return      string  HTML
  */
-if( ! function_exists( 'rtmedia_who_like_html' ) ){
-	function rtmedia_who_like_html( $like_count, $user_like_it ){
+if ( ! function_exists( 'rtmedia_who_like_html' ) ) {
+	function rtmedia_who_like_html( $like_count, $user_like_it ) {
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: add default param value and you can remove lines below.

File: app/main/controllers/template/rtmedia-functions.php [hunk 4]
```diff
@@ -2824,12 +2828,12 @@ function show_rtmedia_like_counts() {
...
 		$like_count = ( $like_count ) ? $like_count : false;
 		$user_like_it = ( $user_like_it ) ? true : false;
 		$like_count_new = $like_count;
-		$html = "";
+		$html = '';
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: align equal signs

File: app/main/controllers/template/rtmedia-functions.php [hunk 5]
```diff
@@ -2824,12 +2828,12 @@ function show_rtmedia_like_counts() {
...
 		if ( $like_count == 1 && $user_like_it ) {
 			/**
 			 * rtmedia you like text
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: Doc comment must start with a capital letter

File: app/main/controllers/template/rtmedia-functions.php [hunk 6]
```diff
@@ -2867,7 +2871,7 @@ function rtmedia_who_like_html( $like_count, $user_like_it ){
 			 * @param int $user_like_it User Like it or Not
 			 * @return html TEXT to  display
 			*/
-			$html .=  apply_filters( 'rtmedia_like_html_othe_likes_this', _n( ' person likes this', ' people like this', $like_count_new, 'buddypress-media' ) ,$like_count, $user_like_it );
+			$html .= apply_filters( 'rtmedia_like_html_othe_likes_this', _n( ' person likes this', ' people like this', $like_count_new, 'buddypress-media' ) ,$like_count, $user_like_it );
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: space after function `_n` and no space before $like_count

File: app/main/controllers/template/rtmedia-functions.php [hunk 7]
```diff
@@ -2877,7 +2881,7 @@ function rtmedia_who_like_html( $like_count, $user_like_it ){
 		 * @param int $user_like_it User Like it or Not
 		 * @return html TEXT to  display
 		*/
-		$html =  apply_filters( 'rtmedia_who_like_html', $html ,$like_count, $user_like_it );
+		$html = apply_filters( 'rtmedia_who_like_html', $html ,$like_count, $user_like_it );
```
Comment (milindmore22) [inline app/main/controllers/template/rtmedia-functions.php]: space and comma at unusual places

**Pull request merged**